apps/judicial-system/web/src/routes/Prosecutor/RestrictionRequest/StepOne/StepOne.tsx

```diff
@@ -46,6 +46,7 @@ export const StepOne: React.FC = () => {
           name: theCase.defendants[0].name,
           address: theCase.defendants[0].address,
           nationalId: theCase.defendants[0].nationalId,
+          noNationalId: theCase.defendants[0].noNationalId,
```

Comment:
Shouldn't citizenship be updated as well?

Reply:
Jup! Thanks 🙏